website/blog/2019-06-04-hello-reason-react-native.md

```diff
@@ -0,0 +1,129 @@
+  vs
+  [`reason-react-native` `Style`](https://github.com/reasonml-community/bs-react-native/blob/958cd4e3a5ffa303304a0b2404cd53b5f49e649f/reason-react-native/src/apis/Style.bs.js)
+- [`bs-react-native` `Platform`](https://github.com/reasonml-community/bs-react-native/blob/958cd4e3a5ffa303304a0b2404cd53b5f49e649f/bs-react-native/src/platform.bs.js)
+  vs
+  [`reason-react-native` `Platform`](https://github.com/reasonml-community/bs-react-native/blob/958cd4e3a5ffa303304a0b2404cd53b5f49e649f/reason-react-native/src/apis/Platform.bs.js)
+
+Some modules are even empty or with some minor `require()` that can be themself
+almost empty & easily removed by some JavaScript bundlers.
+
+How cool is that?
+
+## But why`reason-react-native` ?
+
+`bs-react-native` already had multiples breaking changes in the past & we didn't
+really wanted to offer a poor experience by creating a new release with tons of
+changes that won't be easy to codemod (or would require much more work from us).
+
+On top of that, `reason-react` latest release already added an important
+breaking change with
+[JSX 3](https://reasonml.github.io/reason-react/docs/en/jsx).
+
+That's why we had this idea about offering a easy & smooth transition by
+offering 2 packages to say goodbye to `bs-react-native` without pain.
+
+- `bs-react-native-jsx3-compat`
+- `reason-react-native`
+
+## Wait, what is `bs-react-native-jsx3-compat` ?
+
+`bs-react-native` is using JSX 2 so in order to use it with JSX 3 we would have
+to create bindings for the bindings to offer both JSX 2 & 3 bindings. And since
+we wanted to go away with some current APIs of `bs-react-native` we started to
+think about a new approach.
+
+We know that we wanted to:
+
+- offer to `bs-react-native` users a migration plan, so they can start very
+  easily to use JSX 3 & new `reason-react` (so they can use hooks),
+- offer new `reason-react-native` bindings, more accessible & with a zero cost
+  approach.
+- offer a smooth transition between this 2 packages so people can slowly upgrade
+  their codebase without having to make an insane PR that switch from one to the
+  other.
+
+We don't want your co-workers to hate you with a PR that comes directly from
+hell 😅.
+
+That were we thought about offering `bs-react-native-jsx3-compat`. It's a
+package that have the same APIs has `bs-react-native` but that use
+`reason-react-native` under the hood.
+
+**This means you will have to make a migration from JSX 2 to 3**. That's the
+only thing you will have to do, then with a single change from `bs-react-native`
+with `bs-react-native-jsx3-compat` in your `bs-dependencies` you will be ready
+to use hooks & directly benefit of (almost) zero-cost bindings!
+
+And since we care about you & don't want you to upgrade from JSX 2 to 3 by hand,
+we got you covered with a script that comes right the heart & help you with
```

Comment:
I don't understand "comes right the heart".

Reply:
I missed "from": comes right from the heart. Anyway, switched to "we got you covered with a script made with love to help you with that."